client/components/tabBar.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import classNames from './scss/tabBar.module.scss'
+import getClassNames from "../utilities/getClassnames";
+
+class TabBar extends Component {
+    /*constructor(props){
+        super(props);
+        this.state = {selectedId: 1};
+    }*/
+
+    handleClick = (event) => {
+        const id = event.target.id;
+        //this.setState({selectedId: id});
+        this.props.handleChanged(id);
+    };
+
+    render() {
+        const {tabs, selectedId} = this.props;
+        const {tab} = classNames;
+        const tab_selected = classNames['tab--selected'];
+        const selected = getClassNames({[tab]: true, [tab_selected]: true});
+        const notSelected = getClassNames({[tab]: true, [tab_selected]: false});
```

Comment:
here you can calculate if tab is selected and pass to get classNames method to calculate classes directly in component. So you don't need ternary operator below to choose what class names to use:)